### PR TITLE
Update changelog for 0.2.4.

### PR DESCRIPTION
**Pull request by abuiles**
Started to work on this, can I have some crowdsource help for filling 

```
The following changes are required if you are upgrading from the previous
version:

- Addon Developers
  + No changes required
- Core Contributors 
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,96 @@
...
+- [#3966](https://github.com/ember-cli/ember-cli/pull/3966) increase timeouts, and use mocha’s inheriting config [@ember-cli](https://github.com/ember-cli)
+- [#4032](https://github.com/ember-cli/ember-cli/pull/4032) Update ember-cli-qunit to v0.3.2 [@HeroicEric](https://github.com/HeroicEric)
+- [#4000](https://github.com/ember-cli/ember-cli/pull/4000) use `escape-string-regexp` module [@sindresorhus](https://github.com/sindresorhus)
+- [#3975](https://github.com/ember-cli/ember-cli/pull/3975) included modules is no longer needed [@ember-cli](https://github.com/ember-cli)
+- [#3978](https://github.com/ember-cli/ember-cli/pull/3978) Update broccoli-es6modules [@marcioj](https://github.com/marcioj)
+- [#3976](https://github.com/ember-cli/ember-cli/pull/3976) Relative require [@ember-cli](https://github.com/ember-cli)
+- [#3982](https://github.com/ember-cli/ember-cli/pull/3982) Changed markdown-color blue to bright-blue to be the same on all platforms [@trabus](https://github.com/trabus)
+- [#3983](https://github.com/ember-cli/ember-cli/pull/3983) Revert "Changed markdown-color blue to bright-blue to be the same on all... [@ember-cli](https://github.com/ember-cli)
```
Comment (chadhietala) [inline CHANGELOG.md]: Probably can remove as it was a revert

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+- [#3869](https://github.com/ember-cli/ember-cli/pull/3869) [BUGFIX] Use posix path for in-repo-addons in package.json [@trabus](https://github.com/trabus)
+- [#3846](https://github.com/ember-cli/ember-cli/pull/3846) [BUGFIX] Prevent addon-import blueprint from generating if entity name is undefined [@trabus](https://github.com/trabus)
+- [#3851](https://github.com/ember-cli/ember-cli/pull/3851) Use shorthand ES6 syntax for addon -> app re-exports [@jayphelps](https://github.com/jayphelps)
+- [#3848](https://github.com/ember-cli/ember-cli/pull/3848) writeError now looks for filename as well as file [@wagenet](https://github.com/wagenet)
+- [#3858](https://github.com/ember-cli/ember-cli/pull/3858) move github to normal dependencies to hack around: https://github.com/np... [@ember-cli](https://github.com/ember-cli)
+- [#3856](https://github.com/ember-cli/ember-cli/pull/3856) Use shorthand ES6 re-export for addon-imports as well, which landed in #3690 [@jayphelps](https://github.com/jayphelps)
+- [#3880](https://github.com/ember-cli/ember-cli/pull/3880) Revert "Export return value from Router.map (closes #3676)." [@ember-cli](https://github.com/ember-cli)
```
Comment (chadhietala) [inline CHANGELOG.md]: Remove as it was a revert.

**Comment by chadhietala** [comment]
None of these look like there should be anything for addon developers. I would mention that they should test their addons against 1.13. Addons now ship with ember-try and a new .travis.yml file that runs tests against all of the ember channels thanks to @truenorth.

File: CHANGELOG.md (hunk 1 above)
Comment (abuiles) [inline CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (abuiles) [inline CHANGELOG.md]: Fixed.

**Comment by abuiles** [comment]
Updated accordingly to @trabus and @chadhietala comments, thanks! also @cibernox.

I'll wait for 0.12 to come out, update, squash and release.

**Comment by rwjblue** [comment]
Sorry for the delays, 1.12.0 update PR is here: https://github.com/ember-cli/ember-cli/pull/4065

**Comment by trabus** [comment]
Merged, we're good to go.

**Comment by trabus** [comment]
Docs PR is ready: #4068

**Pull request closed without merging**